### DISCUSSION TITLE
Connecting Priority1Design RFiD system to Bonsai

### DISCUSSION THREAD

**Discussion by Elforion** [bonsai-rx/bonsai#1994] [Q&A] [answered]
Hi
I am trying to connect RFiD antennas I have recieved lately to bonsai for the purpose of proper lab animal identification.
The antennas are a part of the whole system from Priority1Design (pcb with microprocessor + antennas https://www.priority1design.com.au/index.html)
Unfortunately there is no communaction between bonsai and the RFiDs through appropriate COM port and "serialread".
Is it even possible to set it up?

**Comment by bruno-f-cruz** [comment]
Can you add the workflow you are trying to use?
Some of these devices seem to require drivers to be installed, have you completed that step?

**Reply by Elforion** [reply]
Yes indeed, I have installed drivers and everything works when I use dedicated software, I can read and write RFiD tags with no issue.
I haven't attached any workflow since I simply try to read anything from the selected COM port by using "SerialRead", "SerialReadLine". I have tried "CreateSerialPort" aswell. There is nothing on the workflow yet, aside from the single node, with propper port selected. I then look at the visualizer to see if anything pops up.

**Reply by Elforion** [reply, marked as answer]
I have solved the issue. The RFiD tag id consist of 17 digits in total. I had to change the "count" value to 17 in "SerialRead" node.
Thank you for your reply Bruno
